src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -285,9 +295,9 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
       const fiatAmount = bnOrZero(cryptoAmount).times(assetMarketData.price)
 
       setValue(Field.FiatAmount, fiatAmount.toString(), { shouldValidate: true })
-      setValue(Field.CryptoAmount, cryptoAmount.toFixed(), { shouldValidate: true })
+      setValue(Field.CryptoAmount, cryptoAmount.toFixed(asset.precision), { shouldValidate: true })
```

Comment:
This was off, and *is* in fact directly related to this PR even though it doesn't look like it.

Percentage option click wasn't taking into account asset precision, and we were ending up with a value that has possibly more dp than the asset's decimals (e.g: BTC).

This is the reason why trying to input the minimum that's stated still failed validation - one amount had the right amount of decimal places, while the other didn't.